pkgs/development/interpreters/python/default.nix

```
@@ -69,12 +69,13 @@ with pkgs;
               recursivePthLoader
             ;
           };
+          aliases = self: super: lib.optionalAttrs (config.allowAliases or true) (import ../../../top-level/python-aliases.nix lib);
```

Since you changed the aliases file to take two arguments, you need to call it with `self` and `super` now.

```suggestion
          aliases = self: super:
            lib.optionalAttrs (config.allowAliases or true)
              (import ../../../top-level/python-aliases.nix lib self super);
```